index.html

```diff
@@ -4853,6 +4853,73 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to be usable to identify the <a>holder</a> to the <a>verifier</a>.
+        </p>
+        <p>
+Relevant metadata about the <code>holder</code> <a>property</a> is expected
+to be available to the <a>verifier</a>. For example, a <a>holder</a> can
```

Comment:
```suggestion
Relevant metadata about the <code>holder</code> <a>property</a> is expected
to be available to the <a>verifier</a>. For example, a <a>holder</a> can
```

This sentence doesn't improve anything, and implies that relevant metadata is expected. Since there may be no other metadata published anywhere, we should probably just delete it.

Is there something that gets lost if we remove it?

To my mind, if the property value is usable to identify the holder, then the holder understands enough to do so, including any implicit metadata such as how to interpret a particular identifier format.

Reply:
it won't let me accept this change suggestion

Reply:
it does seem like verifier needs to discover holder keys in order to verify things.

Reply:
> [@OR13] it won't let me accept this change suggestion

That might be because there doesn't appear to be any change in the suggested new text from @jandrieu, which I think was meant to delete the quoted existing sentence?

That said... I think this sentence _is_ a problem. Maybe this will help?

```suggestion
Relevant metadata about the entity identified by the value of the
<code>holder</code> <a>property</a> is expected to be available to or
retrievable by the <a>verifier</a>. For example, a <a>holder</a> can
```

Reply:
@OR13 — Any comment on the above?

Reply:
I'm accepting this change, and noting that it applies to every instance of `issuer` as well... the `issuer` identifies an `entity` that performs issuance? the `issuer` is not the `entity` ? This seems like a version of http range 14.

Reply:
`issuer`, `subject`, `holder`, and `verifier` all have range `entity`.

* Yes, `issuer` is an explicitly identified `entity` that issues s VC.
* Yes, `subject` is an explicitly or implicitly identified `entity`, referenced within a VC as the subject of one or more claims.
* Yes, `holder` is an implicitly and usually explicitly identifiable `entity`, to which a VC is issued or transferred.
* Yes, `verifier` is an explicitly identifiable `entity`, which tests whether a VC has been tampered with.

Reply:
This seems potentially much clearer that what we have in the current spec... and it also seems to apply directly to: https://github.com/w3c/vc-data-model/pull/1172

Reply:
@TallTed I agree with all of the ranges you have outlined except for `subject`, which does not need to be an `entity` as the `subject` of a verifiable credential can be anything.

Reply:
I read the current definition of `entity` in the terminology section as something with an independent existence that performs one or more roles in the ecosystem, which would typically be 'holder', 'issuer', 'verifier'. That would mean that a something with an independent existence that is not (capable of) performing any such role, doesn't qualify. Examples include books, dogs, passports, and lots of other stuff. If this interpretation is correct, then any such example cannot be used as a subject in a claim, because they do n't qualify as an entity.

I suggest to have a term (name to be decided, let's call it `X` for now) that refers to something (or someone) that has an independent existence. We don't need to use `entity` for that. It's another concept than what `entity` refers to; `entity` is a specialization of `X`.

`issuer`, `holder`, `verifier` then keep having range `entity`, and `subject` would have range `X`, and since `entity` is a specialization of `X`, any `entity` can still be the subject of a claim, but can then also make claims about an `X` that is not an `entity`.

Reply:
As noted elsewhere, I refer all and sundry to [RDF Concepts, Introduction](https://www.w3.org/TR/rdf11-concepts/#resources-and-statements), where `resource` are clearly stated to be synonymous with `entity`, and both/either may be *anything*, and may be the `subject` of a VC.

In order to perform a role, the given entity/resource must be capable of such actions as are required for that role; for instance, a document probably cannot be an issuer or verifier.